### ISSUE TITLE
Loosing steath for no reason.

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#5611]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Loosing stealth for no reason what so ever.
**Expected behaviour**: Tell us what should happen instead
You don't loose stealth for no reason
**Steps to reproduce the problem**:

1. No idea
2. 
3. 

**Include proofs for this behaviour**

https://youtu.be/gWXuRW-YgKc
As you can see ret paladin is sapped(after the sap he does consecration.)
Boomkin is in blind ( starfall coudn't possibly get me out since it i got out of stealth before blind was over.)
I leave the combat, use stealth and then i loose it for no reason what so ever
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by lkrocek** [comment]
from video it looks like you were affected by druid's starfall

**Comment by Jiaku** [comment]
So, is it getting fixed, since you know the reason?
